packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/routes.jsx.hbs

```diff
@@ -18,20 +24,60 @@ const fallback = <Skeleton height="75vh" width="100%" />
 
 // Create your pages here and add them to the routes array
 // Use loadable to split code into smaller js chunks
+{{#if answers.project.hybrid}}
+const Home = loadable(() => import('@salesforce/retail-react-app/app/pages/home'), { fallback })
+{{else}}
 const Home = loadable(() => import('./pages/home'), {fallback})
+{{/if}}
 const MyNewRoute = loadable(() => import('./pages/my-new-route'))
+{{#if answers.project.hybrid}}
+const PageNotFound = loadable(() => import('@salesforce/retail-react-app/app/pages/page-not-found'))
+{{/if}}
 
 const routes = [
     {
         path: '/',
         component: Home,
         exact: true
     },
+    {{#if answers.project.hybrid}}
+    {
+        path: '/home',
+        component: Home,
+        exact: true
+    },
+    {{/if}}
     {
         path: '/my-new-route',
         component: MyNewRoute
     },
+    {{#if answers.project.hybrid}}
+    // remove routes from the base template that we want to go to SFRA
+    ..._routes.filter((route) => {
+        return !(route.path === '/cart' || route.path === '/checkout' || route.path === '*')
+    }),
+    {
+        path: '*',
+        component: withRouter((props) => {
```

Comment:
nit: usually components are defined elsewhere, rather than in-line in the routes file.

Reply:
Good point.

The approach taken here mimics what we've document in https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/phased-headless-rollouts.html but we could touch this up so that components are defined where they should be